rust/src/smb/log.rs

```diff
@@ -136,6 +139,11 @@ fn smb_common_header(jsb: &mut JsonBuilder, state: &SMBState, tx: &SMBTransactio
 
     match tx.type_data {
         Some(SMBTransactionTypeData::SESSIONSETUP(ref x)) => {
+            let mut outbuf: [u8; 16] = [0; 16];
+            if let Ok(_) = smb_get_fingerprint(&state, tx, &mut outbuf) {
+                jsb.set_string("fingerprint", &to_hex(&outbuf[..]))?;
```

Comment:
Need to add this to the `eve.json` schema?

Reply:
Indeed, force-pushed a change to get CI running again